autogluon/task/tabular_prediction/predictor.py

```diff
@@ -688,6 +688,75 @@ def load_data_internal(self, dataset='train', return_X=True, return_y=True):
         y = load_y() if return_y else None
         return X, y
 
+    def reduce_memory_size(self, remove_data=True, remove_fit_stack=True, requires_save=True, reduce_children=False):
```

Comment:
should this method be called reduce_memory_size? I'd consider calling it something like `save_space()` since it's equally about saving disk-space.  More importantly, users will need to understand that this is different than say model compression techniques via refit_full to collapse-bags or (future) ensemble-pruning.

Reply:
renamed to save_space(). Added clarification in documentation about impact.